forge-game/pom.xml

```diff
@@ -24,9 +28,9 @@
             <version>1.2</version>
         </dependency>
         <dependency>
-            <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
-            <version>7.4.0</version>
+            <groupId>org.junit.jupiter</groupId>
```

Comment:
I don't understand your reasoning, you switched us mostly to TestNG here: c7d0dde
we already have a strong dependency on it because of PowerMock
why would we need a second framework again? 👎

Reply:
(Replied below.)